### DISCUSSION TITLE
How to use SimpleLogin API to access API endpoints using curl

### DISCUSSION THREAD

**Discussion by AedenThomas** [simple-login/app#533] [Q&A]
How do I create a new alias from SimpleLogin's API using bash (curl)?

**Comment by nguyenkims** [comment]
The API doc is on https://github.com/simple-login/app/blob/master/docs/api.md that can be converted to curl syntax I think.

**Reply by AedenThomas** [reply]
Hope SimpleLogin can make [docs easy like AnonAddy](https://app.anonaddy.com/docs/#create-new-alias) for beginners like me

**Reply by hexclann** [reply]
@AedenThomas Did you able to access the data using curl? I'm getting wrong api key every time

**Reply by hexclann** [reply]
@nguyenkims I have created an API key from the dashboard, but when I make a request I get:
`{"error":"Wrong api key"}` error despite copy pasting the exact key from the UI. Im getting this error on both selfhosted and simplelogin.io site

This is the command:
`curl -s -X GET "https://app.mydomain.com/api/v5/alias/options" -H "Authorization: Bearer gcthyhtdhffqmthrrroRedactedToken" -H "Content-Type:application/json"`
